File: B/C#/b010.cs
using System;
namespace Solution {
    class Solution {
        static void Main(string[] args) {

            const int center = 55;
            const int eleven = 11;
            int[] XA = new int[eleven];
            int[] XB = new int[eleven];

            string[] tmp = System.Console.ReadLine().Trim().Split(' ');
            string T = String.Copy(tmp[0]);
            int    U = Int32.Parse(tmp[1]);

            tmp = System.Console.ReadLine().Trim().Split(' ');
            for (int i=0; i<eleven; i++) { XA[i] = Int32.Parse(tmp[i]); }
            tmp = System.Console.ReadLine().Trim().Split(' ');
            for (int i=0; i<eleven; i++) { XB[i] = Int32.Parse(tmp[i]); }

            if (T == "A")
            {
                Array.Sort(XB);
                int b = XB[9];
                int pass = XA[U-1];
                int num = 1;
                bool flg = false;
                for (int i=0; i<eleven; i++)
                {
                    int a = XA[i];
                    if (a>=center && a>b && a>pass)
                    {
                        System.Console.WriteLine(num);
                        flg = true;
                    }
                    num += 1;
                }
                if (flg == false) { System.Console.WriteLine("None"); }
            } else {
                Array.Sort(XA);
                int a = XA[1];
                int pass = XB[U-1];
                int num = 1;
                bool flg = false;
                for (int i=0; i<eleven; i++)
                {
                    int b = XB[i];
                    if (b<=center && b<a && b<pass)
                    {
                        System.Console.WriteLine(num);
                        flg = true;
                    }
                    num += 1;
                }
                if (flg == false) { System.Console.WriteLine("None"); }
            }
        }
    }
}

File: B/JavaScript/b010.js
process.stdin.resume();
process.stdin.setEncoding('utf8');
process.stdin.on('data', function (chunk) {

    var center = 55;
    var eleven = 11;
    var XA = new Array(eleven);
    var XB = new Array(eleven);
    var tmp = [];
    var a,b,pass,num,flg,i;

    var lines = chunk.toString().split('\n');
    var s = lines[0].replace(/(^\s+)|(\s+$)/g, "").split(" ");
    var T = s[0];
    var U = Number(s[1]);

    tmp = lines[1].replace(/(^\s+)|(\s+$)/g, "").split(" ");
    for (i=0; i<eleven; i++) { XA[i] = Number(tmp[i]); }
    tmp = lines[2].replace(/(^\s+)|(\s+$)/g, "").split(" ");
    for (i=0; i<eleven; i++) { XB[i] = Number(tmp[i]); }

    if (T == "A") {
        XB = XB.sort(function(a,b){ return a -b; });
        b = XB[9];
        pass = XA[U-1];
        num = 1;
        flg = false;
        for (i=0; i<eleven; i++) {
            a = XA[i];
            if (a>=center && a>b && a>pass) {
                console.log(num);
                flg = true;
            }
            num += 1;
        }
        if (flg === false) { console.log("None"); }
    } else {
        XA = XA.sort(function(a,b){ return a -b; });
        a = XA[1];
        pass = XB[U-1];
        num = 1;
        flg = false;
        for (i=0; i<eleven; i++) {
            b = XB[i];
            if (b<=center && b<a && b<pass) {
                console.log(num);
                flg = true;
            }
            num += 1;
        }
        if (flg === false) { console.log("None"); }
    }
});
